### ISSUE TITLE
Keyboard Error

### ISSUE THREAD

**Issue by brgkdm** [brgkdm/KeyboardFX#1]
# **Keyboard Error**

Hello, when you press any key on the physical keyboard, the highlighting on the virtual keyboard does not occur and nothing appears.
 
I am definitely aware that there is a problem, so I am asking for help from those who want to play around and have fun.
 
If you have any questions or want to know something, you can contact me at any time. 

Thank you.

**Comment by FelipeLohan** [comment]
Hello! im helping u, im forked your project (;

**Comment by FelipeLohan** [comment]
the class "input-display" dont exists in html.

**Comment by brgkdm** [comment]
> the class "input-display" dont exists in html.

Yes, I am aware of this, but when I add it, instead of highlighting the keys, the keys we press are written on one side of the page. This is where the problem arose.

**Comment by mohamedhesham221** [comment]
Hi, I'm helping you now

**Comment by brgkdm** [comment]
see: https://github.com/brgkdm/KeyboardFX/pull/2
